src/storiesStore/selectors.js

```diff
@@ -0,0 +1,6 @@
+export function getStories(state) {
+  return state.stories;
+}
+export function getIsLoading(state) {
+  return state.isLoading;
+}
```

Comment:
This is nice and let's leave it. But try to only add selectors as you need them.